region/norfair/east/Double Chamber.json

```diff
@@ -960,6 +961,20 @@
         {"heatFrames": 200}
       ]
     },
+    {
+      "link": [5, 3],
+      "name": "Crystal Flash Clip",
+      "requires": [
+        "h_heatProof",
+        "h_canJumpIntoCrystalFlashClip",
+        "Grapple"
+      ],
+      "note": [
+        "Stand on the second highest platform and jump into a Crystal Flash.",
```

Comment:
are you jumping against the right wall - through the broken shot block?

Reply:
I wasn't aligned with anything; but under the broken shot block yeah.  A bombIntoCF isn't high enough so you need a normal jump morph.